pkg/ccl/changefeedccl/changefeed_stmt.go

```diff
@@ -904,6 +904,18 @@ func validateSink(
 	if err != nil {
 		return err
 	}
+	u, err := url.Parse(details.SinkURI)
+	if err != nil {
+		return err
+	}
+
+	if u.Scheme == changefeedbase.DeprecatedSinkSchemeHTTP || u.Scheme == changefeedbase.DeprecatedSinkSchemeHTTP {
+		p.BufferClientNotice(ctx, pgnotice.Newf(
+			`%s sinks will emit using cloud storage semantics. For a webhook sink, prepend webhook- to the sink URI. Prepend file- to remove this notice.`,
```

Comment:
maybe rephrase a bit to avoid using "prepend"? Something like "Use webhook-https for webhook semantics, use file-http:// .."

Reply:
Done.